template/python/Dockerfile

```diff
@@ -1,27 +1,34 @@
 FROM python:2.7-alpine
 
+RUN adduser app -D
+
 # Alternatively use ADD https:// (which will not be cached by Docker builder)
 RUN apk --no-cache add curl \ 
     && echo "Pulling watchdog binary from Github." \
     && curl -sSL https://github.com/openfaas/faas/releases/download/0.6.1/fwatchdog > /usr/bin/fwatchdog \
     && chmod +x /usr/bin/fwatchdog \
     && apk del curl --no-cache
 
-WORKDIR /root/
+RUN mkdir -p /home/app/function
```

Comment:
What's going on with the ordering here? Does it optimize the build?

Reply:
This is the same ordering used in the node template, which I am basing these templates off of.